dynamic_instrumentation/src/main.rs

```diff
@@ -0,0 +1,413 @@
+#![feature(rustc_private)]
+extern crate rustc_driver;
+extern crate rustc_session;
+extern crate rustc_span;
+
+use std::{
+    env,
+    ffi::OsString,
+    hash::Hash,
+    io::ErrorKind,
+    path::{Path, PathBuf},
+    process::{self, Command, ExitStatus},
+    str::FromStr,
+};
+
+use c2rust_dynamic_instrumentation::{MirTransformCallbacks, INSTRUMENTER};
+use camino::Utf8Path;
+use cargo_metadata::{Metadata, MetadataCommand, Package, Target};
+use clap::Parser;
+use color_eyre::eyre;
```

Comment:
I don't think we should be mixing anyhow and eyre just to get console colors. If we want to switch to eyre from anyhow, that should be a separate change.

Reply:
Okay, I can switch things back to `anyhow` for now and do the change to `eyre` separately.  `eyre` I find useful mostly for the backtraces, though the colors are also very nice.  It's a shame it's not compatible with `anyhow`, as it is a fork of `anyhow` and very similar.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/554/commits/045c62d7ada40b73fa67aa6521e8d62dc1185073.